docs/data/components/select/select.mdx

```diff
@@ -0,0 +1,349 @@
+- `<Select.Arrow />` renders the select popup's arrow when using `alignMethod="trigger"`.
+
+```jsx
+<Select.Root>
+  <Select.Trigger>
+    <Select.Value />
+    <Select.Icon />
+  </Select.Trigger>
+
+  <Select.Backdrop />
+
+  <Select.Positioner>
+    <Select.ScrollUpArrow />
+
+    <Select.Popup>
+      <Select.Group>
+        <Select.GroupLabel />
+        <Select.Option>
+          <Select.OptionText />
+          <Select.OptionIndicator />
+        </Select.Option>
+      </Select.Group>
+      <Select.Separator />
+    </Select.Popup>
+
+    <Select.ScrollDownArrow />
+
+    <Select.Arrow />
+  </Select.Positioner>
+</Select.Root>
+```
+
+## Default value
+
+To set an initial value when uncontrolled, use `defaultValue`:
+
+```jsx
+<Select.Root defaultValue="system">
+```
+
+## Empty value
+
+The select's value is empty (`null`) by default, which enables an empty `Option` to be initially selected when it has no `value` prop:
+
+<Demo demo="SelectEmpty" defaultCodeOpen="false" />
+
+## Controlled
+
+To control the value with external state, specify the `value` and `onValueChange` props:
+
+```jsx
+const [value, setValue] = React.useState('system');
+
+return (
+  <Select.Root value={value} onValueChange={setValue}>
+    {/* subcomponents */}
+  </Select.Root>
+);
+```
+
+## Option indicator
+
+The `Select.OptionIndicator` subcomponent renders an indicator inside an option to indicate it's selected. By default, it renders a check icon, but this can be customized:
+
+```jsx
+<Select.OptionIndicator>
+  <MyCheckIcon />
+</Select.OptionIndicator>
+```
+
+The `[data-selected]` attribute is added to the subcomponent when its owning option is selected. This is necessary to hide the indicator when the option is not selected:
+
+```css
+.SelectOptionIndicator {
+  visibility: hidden;
+}
+
+.SelectOptionIndicator[data-selected] {
+  visibility: visible;
+}
+```
+
+## Grouped options
+
+`Select.OptionGroup` can be used to group options together with a label. The `Select.OptionGroupLabel` subcomponent renders the label:
+
+```jsx
+<Select.OptionGroup>
+  <Select.OptionGroupLabel>Label</Select.OptionGroupLabel>
+  <Select.Option value="option1">Option 1</Select.Option>
+  <Select.Option value="option2">Option 2</Select.Option>
+</Select.OptionGroup>
+```
+
+<Demo demo="SelectGroup" defaultCodeOpen="false" />
+
+## Align method
+
+Two different methods to align the popup are available:
+
+- `alignMethod="item"` (default)
+- `alignMethod="trigger"`
+
+```jsx
+<Select.Root alignMethod="trigger">
+```
+
+- **`item`**: aligns the popup such that the selected option inside of it appears centered over the trigger. If there's not enough space, it falls back to `trigger` anchoring. This method is useful as it allows the user to select the an option in a single click or "pointer cycle" (pointer down, pointer move, pointer up). This is the native behavior on macOS. The scroll arrow components must be used to ensure this is the case.
```

Comment:
Doesn't the "trigger" option allow selecting with a single click?

Reply:
Yes but only if there's no scrollable content